### PR TITLE
Fix pageNav in tweaking the page structure

### LINKED ISSUE #1097: Incomplete pageNav in tweaking the page structure page
* **MarkBind Version:**
Latest master

**What did you do? Please include the actual source code causing the issue.**
N.A.

**What did you expect to happen?**
Expected `filterTags` to be displayed in the pageNav.

**What actually happened? Please include the actual, raw output.**
`filterTags` is not displayed in the pageNav. 

Screen Shot 2020-03-08 at 5 46 08 AM

### PR DESCRIPTION
**Pull request by Tejas2805**
Fixes #1097 
**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Documentation update
• [X] Bug fix

**What is the rationale for this request?**
The pageNav is missing filter tags. 

**What changes did you make? (Give an overview)**
Updated the pageNav for `tweaking the page structure` to show filter tags too. 

**Provide some example code that this change will affect:**
N.A.

**Is there anything you'd like reviewers to focus on?**
N.A.

**Testing instructions:**
Check the preview. 

**Proposed commit message: (wrap lines at 72 characters)**
```
The pageNav for tweaking the page structure is missing filter tags.

Let's fix the headings in filter tags and the pageNav so that pageNav
display filter tags appropriately.
```

### CODE REVIEW

**Comment by Tejas2805** [comment]
@damithc Can I get your review on this?

File: docs/userGuide/plugins/filterTags.mbdf [hunk 1]
```diff
@@ -1,7 +1,9 @@
-#### `filterTags`: Toggling alternative contents in a page
+## Filter tags
```
Comment (damithc) [inline docs/userGuide/plugins/filterTags.mbdf]: `Filter tags` is a new term that is not mentioned anywhere else and not a standard term either.

Perhaps `Tags`?

**Comment by Tejas2805** [comment]
@marvinchin Can I get your review on this?

File: docs/userGuide/plugins/filterTags.mbdf [hunk 2]
```diff
@@ -1,7 +1,9 @@
-#### `filterTags`: Toggling alternative contents in a page
+## Tags
```
Comment (marvinchin) [inline docs/userGuide/plugins/filterTags.mbdf]: Should we rename the file as well?

Comment (Tejas2805) [reply docs/userGuide/plugins/filterTags.mbdf]: Yes! Will do.

**Pull request merged**